internal/gen/controller/api/services/auth_method_service_taggable_test.go

```diff
@@ -1,148 +0,0 @@
-package services_test
-
-import (
-	"context"
-	"encoding/json"
-	"testing"
-	"time"
-
-	"github.com/hashicorp/boundary/internal/gen/controller/api/services"
-	"github.com/hashicorp/boundary/sdk/pbs/controller/api"
-	"github.com/hashicorp/boundary/sdk/wrapper"
-	"github.com/hashicorp/eventlogger"
-	"github.com/hashicorp/eventlogger/filters/encrypt"
-	"github.com/stretchr/testify/assert"
-	"github.com/stretchr/testify/require"
-	structpb "google.golang.org/protobuf/types/known/structpb"
-)
-
-// TestAuthenticate_Tags will test that the response filtering aligns with the
-// AuthenticateResponse and AuthenticateResponse tags.  See:
-// internal/tests/api/authmethods/authenticate_test.go TestAuthenticate where
-// the audit events produced using these tags is unit tested.
-func TestAuthenticate_Tags(t *testing.T) {
```

Comment:
I think we should be able to keep this test in place. I would potentially rename the file, but these tests should be asserting that the classification is correct, we should probably add similar tests for the other packages.